sdk/core/System.ServiceModel.Rest/api/System.ServiceModel.Rest.net461.cs

```diff
@@ -0,0 +1,62 @@
+namespace System.ServiceModel.Rest
+{
+    public abstract partial class Credential
+    {
+        protected Credential() { }
+    }
+    public partial class KeyCredential : System.ServiceModel.Rest.Credential
+    {
+        public KeyCredential(string key) { }
+        public string Key { get { throw null; } }
+        public void Update(string key) { }
+    }
+    public partial class NullableResult<T>
+    {
+        public NullableResult(T? value, System.ServiceModel.Rest.Result result) { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public virtual bool HasValue { get { throw null; } }
+        public virtual T? Value { get { throw null; } }
+        public virtual System.ServiceModel.Rest.Result GetRawResult() { throw null; }
+    }
+    public abstract partial class PipelineOptions
+    {
+        protected PipelineOptions() { }
+    }
+    public abstract partial class RequestBody : System.IDisposable
+    {
+        protected RequestBody() { }
+        public abstract void Dispose();
+        public abstract bool TryComputeLength(out long length);
+        public abstract void WriteTo(System.IO.Stream stream, System.Threading.CancellationToken cancellation);
+        public abstract System.Threading.Tasks.Task WriteToAsync(System.IO.Stream stream, System.Threading.CancellationToken cancellation);
+    }
+    public partial class RequestOptions : System.ServiceModel.Rest.PipelineOptions
```

Comment:
Can we just have PipelineOptions (and add CancellationToken to it)

Reply:
Yep, updated